Auto-publishing the TypeScript package fails

npm ERR! code E403
npm ERR! 403 403 Forbidden - PUT https://registry.npmjs.org/eslint-config-motley-typescript - Two-factor authentication is required to publish this package but an automation token was specified
npm ERR! 403 In most cases, you or one of your dependencies are requesting
npm ERR! 403 a package version that is forbidden by your security policy, or
npm ERR! 403 on a server you do not have access to.

Weird 🤔

--- a/.github/workflows/publish.yml
+++ b/.github/workflows/publish.yml
@@ -19,7 +19,7 @@ jobs:
         cache: 'npm'
     - run: npm ci
     - run: npm run test:ci
-  publish-eslint-config-motley:
+  publish-packages:
     runs-on: ubuntu-latest
     needs: build-and-test
     steps:
@@ -33,16 +33,6 @@ jobs:
       with:
         token: ${{ secrets.NPM_TOKEN }}
         package: './packages/eslint-config-motley/package.json'
-  publish-eslint-config-typescript:
-    runs-on: ubuntu-latest
-    needs: build-and-test
-    steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js 16.x
-      uses: actions/setup-node@v2
-      with:
-        node-version: 16.x
-        cache: 'npm'
     - uses: JS-DevTools/npm-publish@v1
       with:
         token: ${{ secrets.NPM_TOKEN }}

